index.html

```diff
@@ -1854,18 +1856,16 @@ <h3>The <dfn>NFCReaderOptions</dfn> dictionary</h3>
           </li>
           <li>
             If there is no underlying <a>NFC Adapter</a>, or if a connection cannot
-            be established, then reject <var>p</var> with a <code>"<a>NotReadableError</a>"</code>
-            <code><a>DOMException</a></code> and return <var>p</var>.
+            be established, then reject <var>p</var> with a <code><a>NotReadableError</a></code>
+            and return <var>p</var>.
```

Comment:
This seems like a regression to me, FWIW.

Reply:
This fits with now it is done in WHATWG URL spec

Reply:
![image](https://user-images.githubusercontent.com/1101339/53119477-e7410f80-354f-11e9-95e0-59a0248d68b2.png)

Or you want to make clear that it is a DOMException when used with a promise and not just thrown?

Reply:
A `TypeError` is not a `DOMException`, it's a separate class. Look at https://fetch.spec.whatwg.org/ for a specification that throws both `TypeError` and "`AbortError`" `DOMException`.